KPI-team1-frontend/tsconfig.json

```diff
@@ -16,8 +16,8 @@
 
     /* Linting */
     "strict": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
```

Comment:
Ideally I think we would want those flags to be true ;) - could you maybe add a todo comment here so that we remember to enable them again?